create-toolchain

```diff
@@ -98,11 +94,15 @@ function link_file {
 }
 export -f link_file
 
+info "Modifying Linux toolchain"
 # symlink all files in these dirs from the host toolchain...
 find linux/host \( -path 'linux/host/bin/*' -o -path 'linux/host/local/*' -o -path 'linux/host/share/*' -o -path 'linux/host/lib/*' \) -and -not -type d -print0 | xargs -0 -I{} bash -c 'link_file "$@"' _ {}
 # ...but replace some components with mac ones...
 rm -rf linux/iphone/lib/swift{,_static}
-cp -a "${mac_toolchain}"/usr/lib/{swift,swift-5.0,swift-5.5,arc} linux/iphone/lib/
+if [[ "${is_mac}" -eq 1 ]]; then
+	cp -a "${mac_toolchain}"/usr/lib/{swift-5.0,arc} linux/iphone/lib/
+fi
```

Comment:
What's the rationale behind this change? arclite is arguably less relevant now, but including swift-5.0 is still necessary if back-deploying to iOS < 12.2.

Reply:
They're not included in the OSX toolchain provided by Apple for Ubuntu -- https://github.com/L1ghtmann/swift-toolchain-linux/actions/runs/3506320032/jobs/5873283058#step:3:5360

Reply:
hmm interesting, thanks for flagging — in that case we should either emit a warning that the toolchain won't include pre-12.2 compat, or not support building on Linux. There's also a third option which is getting the 5.0 libs from somewhere else (since they're effectively frozen in time now) but I'm not sure what a good official source would be.

Reply:
re the third option, we might be able to extract them from https://download.swift.org/swift-5.0-release/xcode/swift-5.0-RELEASE/swift-5.0-RELEASE-osx.pkg

Reply:
Unsure which 5.0 directories need to be included, so just went with a warning for the time being.